kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/Environment.java

```diff
@@ -77,6 +78,7 @@ private Environment() {
      */
     private static final String SKIP_TEARDOWN_DEFAULT = "false";
     private static final String STRIMZI_FEATURE_GATES_DEFAULT = "";
+    private static final String STRIMZI_NAMESPACE_DEFAULT = Constants.KAFKA_DEFAULT_NAMESPACE;
```

Comment:
nit: I don't think there is any value in this constant given its just a re-assignment of an existing constant.